server/service/src/sync/translations/invoice.rs

```diff
@@ -354,10 +357,18 @@ impl SyncTranslation for InvoiceTranslation {
             prescriber_ID: clinician_id,
         };
 
+        let json_recod = serde_json::to_value(&legacy_row)?;
```

Comment:
Hmm, I wonder how this sneaked in ?

Reply:
huh. no idea!

Reply:
ah. come in from `main`. we should probably have updated `develop` earlier.
it's been a long time between releases. Would be good to have these changes in develop tho